### PR TITLE
added skie/PhpStorm link

### PR DESCRIPTION
**Pull request by skie**
https://github.com/skie/PhpStorm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -365,6 +366,11 @@ Additional lists you might find useful:
 - [Puppet](https://puppetlabs.com/) - A server automation framework and application.
 - [Vagrant](https://www.vagrantup.com/) - A portable development environment utility.
 
+## IDE  extensions
+*Software and tools to integrate with cakephp specific features.*
+
+- [Command line tool console gneration plugin for PhpStorm](https://github.com/skie/PhpStorm) - Builds xml file to use autocomplete for cakephp console from PhpStorm IDE.
```
Comment (dereuromark) [inline README.md]: `XML` and `CakePHP`

Comment (dereuromark) [inline README.md]: Please follow the naming scheme as per Contribution.md

**Comment by bravo-kernel** [comment]
Missing e in `gneration`

**Comment by dereuromark** [comment]
IMO this does not need its own category, putting it into the existing "Development Env" one would totally suffice.

File: README.md (hunk 1 above)
Comment (jadb) [inline README.md]: For description, how about:

```
[PhpStorm plugin](...) - CakePHP console auto-completion for the PhpStorm IDE
```

**Comment by skie** [comment]
There is plugin usage exampleexample

**Comment by jadb** [comment]
@skie nice! 

Would you mind just bumping it a little up in the list under that category so it's alphabetically sorted?

**Comment by dereuromark** [comment]
The word Console looks wrong to me in that sentence

**Comment by jadb** [comment]
@dereuromark why? What do you suggest?

**Comment by skie** [comment]
The tool name is "Command line Tools Console"
So console is correct here.

**Comment by dereuromark** [comment]
The specifics and that it is a shell is irrelevant here IMO, so it should read what it does:

    CakePHP auto-completion support for the PhpStorm IDE.

or sth like that.

**Comment by jadb** [comment]
@dereuromark I will have to disagree.

I see it provides shell tasks not auto-completion of code snippets (which is what your description makes me think of).

**Comment by dereuromark** [comment]
Then I misread the plugin itself, so it does not help with autocompletion of the code, but the console commands there. Well, then I take my last comment back,of course. Sry for that!

    CakePHP auto-completion support for console command in PhpStorm IDE.

Is that clearer maybe?

**Comment by skie** [comment]
Final update added

**Comment by dereuromark** [comment]
Truly awesome, thank you!

**Pull request merged**